faq.html

```diff
@@ -155,10 +155,13 @@ <h2 id="general">General</h2>
       <a href="{{page.root}}/img/software-carpentry-banner.png">Software Carpentry logo</a>
       are both trademarked,
       and may only be used with our explicit prior permission.
-      We normally grant this to any training course that covers our
-      <a href="#core-topics">core topics</a>
-      has at least one <a href="{{page.root}}/pages/team.html">certified instructor</a> teaching,
-      and sends us summary information about attendees.
+      We normally grant this to any training course that:
```

Comment:
On Tue, Mar 17, 2015 at 10:20:25AM -0700, Raniere Silva wrote:
> Can we replace
> https://github.com/swcarpentry/site/blob/master/bin/preprocess.py
> and
> https://github.com/swcarpentry/site/blob/master/config/workshops.yml
> with queries to `amy`?

Sure.  It looks like the logic we want to replace is actually in
https://github.com/swcarpentry/site/blob/master/bin/workshops.py.  I'm
happy working up a pull request that hits amy for this data, asking
for a valid username/password to authorize the amy request.